### PR TITLE
Close #24, Fix `zone_awareness_config` config block

### LINKED ISSUE #24: Fix `zone_awareness_config` config part
Hi there,

I have an issue with `zone_awareness_config`,

this parameter's nested `availability_zone_count` can either be set to `2` or `3` or omitted at all. I can't set it to one.
So if I set `zone_awareness_enabled = false` and `availability_zone_count = 2`

Every time I run terraform plan I get this

```
      ~ cluster_config {
            dedicated_master_count   = 0
            dedicated_master_enabled = false
            instance_count           = 1
            instance_type            = "t2.small.elasticsearch"
            zone_awareness_enabled   = false

          + zone_awareness_config {
              + availability_zone_count = 2
            }
        }
```

Expected behaviour - If `availability_zone_count` is set to `1`, omit this config part at all.

I'm looking into what I can done to fix it and probably will send a pull-request later on.

### PR DESCRIPTION
**Pull request by netflash**
Fixes #24

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -126,8 +140,11 @@ resource "aws_elasticsearch_domain" "default" {
     dedicated_master_type    = var.dedicated_master_type
     zone_awareness_enabled   = var.zone_awareness_enabled
 
-    zone_awareness_config {
-      availability_zone_count = var.availability_zone_count
+    dynamic "zone_awareness_config" {
+      for_each = null_resource.azs[*].triggers
+      content {
+        availability_zone_count = zone_awareness_config.value.availability_zone_count
+      }
```
Comment (leoblanc) [inline main.tf]: Nice!

**Comment by maxime-michel** [comment]
@aknysh could you please approve the README changes so the PR can be merged?

File: main.tf [hunk 2]
```diff
@@ -126,8 +140,11 @@ resource "aws_elasticsearch_domain" "default" {
     dedicated_master_type    = var.dedicated_master_type
     zone_awareness_enabled   = var.zone_awareness_enabled
 
-    zone_awareness_config {
-      availability_zone_count = var.availability_zone_count
+    dynamic "zone_awareness_config" {
+      for_each = null_resource.azs[*].triggers
```
Comment (aknysh) [inline main.tf]:
```suggestion
      for_each = var.availability_zone_count > 1 ? ["true"] : []
```

it could be done w/o using `null_resource`

**Pull request merged**